encryption_works.md

```diff
@@ -339,53 +339,43 @@ Snowden was right to be cautious and to insist that he check Gellman's new PGP f
 
-You can get started using Tails by [downloading](https://tails.boum.org/download/index.en.html) the DVD image and burning it to a DVD. You then need to boot to this DVD. This step is different depending on what model computer you have, but it often involves entering your BIOS and changing your boot order so your computer tries booting from DVD before it tries your hard drive. On newer PCs you might need to disable [UEFI "secure boot"](https://en.wikipedia.org/wiki/Unified_Extensible_Firmware_Interface#Booting) in the BIOS as well, which is the crypto that's used to make sure your computer will only boot to digitally signed versions of Windows (which, in affect, makes it harder for people to boot into non-Windows operating systems). The [Tails website](https://tails.boum.org/download/index.en.html#start) has more information on booting Tools from a DVD or USB stick.
+Since Tails isn't a practical choice for daily computer use, setting up Tor, PGP, and OTR encryption on your main operating system is a good idea. 
 
-After booting to the DVD you have the option to install Tails on a USB stick, which is especially useful because it allows you to configure a [persistent volume](https://tails.boum.org/doc/first_steps/persistence/index.en.html), an encrypted section of your USB stick to store your data. Despite starting from a clean slate each time you boot up, it's important for you to be able to have access to your OTR and PGP keys, your Claws Mail (more below) and Pidgin settings, and any documents you're working with. Your persistent volume allows you to do this.
+Every time you boot Tails you start from a clean state. Anything and everything you did previously gets erased and the system reverts back to its default out-of-the-box state. It's designed this way so that if you get infected with malware while using Tails, it's gone the next time you boot up. 
 
-### PGP and Email in Tails
+It's a good idea to install Tails on a fresh storage device (either a DVD or a USB stick) to ensure everything works as it should. Then, follow the instructions on [tails.boum.org](https://tails.boum.org/download/index.en.html)'s website. It's an involved process that involves knowledge of the command-line, but it's worth sticking it out. This is the hardest crypto tool to install, but it is currently the gold standard for anonymously using a  computer, particularly when coupled with PGP-encrypted e-mail with Claws Mail, the e-mail application bundled with Tails. For set-up instructions, check out [Claws](http://www.claws-mail.org/documentation.php?section=general) documentation.
```

Comment:
An experienced user can use the 'dd' command to image the ISO to a block/storage device, but agreed we shouldn't mention it.